google-home-community.groovy

```diff
@@ -1702,6 +1703,50 @@ private deviceTraitPreferences_StartStop(deviceTrait) {
     }
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private deviceTraitPreferences_StatusReport(deviceTrait) {
+    section("Status Report Settings") {
+        input(
+            name: "${deviceTrait.name}.numStatusTypes",
+            title: "Number of Status Types",
+            type: "number",
+            range: "1..${STATUS_REPORT_MAX_TYPES}",
+            defaultValue: 1,
+            required: true,
+            submitOnChange: true
+        )
+    }
```

Comment:
Instead of having to enter the number explicitly I think it would make more sense to use an `href` or `button` to create new entries, like the Toggles trait or global PIN codes do it.